Apps/WebClient/src/Server/Models/UserNote.cs

```diff
@@ -0,0 +1,151 @@
+//-------------------------------------------------------------------------
```

Comment:
Question:  Why repeat the DB model for both Notes and Comments?  Why not add the utility classes into the services?  Not sure I see a lot of value unless the annotations interact with the controller return data?

Reply:
The main reason to have them separate now is because they dont represent the same data anymore. Once is encrypted (db model) while the other is a user representation. Having them separate makes a clear distinction between them and gives us the flexibility to further change each of them individually and map them at the object level.